google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -862,6 +927,10 @@ def get
         def listen &callback
           raise ArgumentError, "callback required" if callback.nil?
 
+          if limit_type == :last
+            raise "Queries that include limit_to_last constraints cannot be streamed. Use Query#get instead."
```

Comment:
See [Java impl](https://github.com/schmidt-sebastian/java-firestore/blob/0a5deea81a1ae907015daff52fa14588c00f76e8/google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java#L1115-L1118)

Reply:
As stated above, this limitation is for Query.stream, not Query.onSnapshot. You should be able to flip the direction of the snapshot for the Listen API.

Reply:
Updated, thank you.